raycicmd/bk_pipeline.go

```diff
@@ -82,6 +82,9 @@ func makeRayWindowsDockerPlugin(config *stepDockerPluginConfig) map[string]any {
 		"shm-size":       "2.5gb",
 		"mount-checkout": true,
 		"environment":    envs,
+		"volumes": []string{
+			"\\\\.\\pipe\\docker_engine:\\\\.\\pipe\\docker_engine",
```

Comment:
you can use backquote "`", to quote the string, and you would not need so many backslashes.